test/cli_checkpoint_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/alibaba/pouch/test/util"
+	"github.com/gotestyourself/gotestyourself/icmd"
+
+	"github.com/go-check/check"
+)
+
+// PouchCheckpointSuite is the test suite for container create API.
+type PouchCheckpointSuite struct{}
+
+func init() {
+	check.Suite(&PouchCheckpointSuite{})
+}
+
+// SetUpTest does common setup in the beginning of each test.
+func (suite *PouchCheckpointSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	SkipIfFalse(c, environment.IsCRIUExist)
+}
+
+// TestCreate tests create checkpoint
+func (suite *PouchCheckpointSuite) TestCheckpointCreate(c *check.C) {
```

Comment:
Could we add one test case `TestCheckpointCreateWithExitFlag` to checkpoint and stop one container, then check the `stopped` status and the checkpoint of this stopped container?

Reply:
already add.